content/influxdb/v2.1/notebooks/downsample.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Downsample data with notebooks
+description: >
+  Downsample data to summarize data at specified time intervals and reduce the
+  overall disk usage as data collects over time.
+  Learn how to create a notebook that downsamples data.
+weight: 104
+influxdb/v2.1/tags: [notebooks]
+menu:
+  influxdb_2_1:
+    name: Downsample data
+    identifier: notebooks-downsample
+    parent: Notebooks
+---
+
+Downsample data to summarize data at specified time intervals and reduce the
+overall disk usage as data collects over time.
+Learn how to create a notebook that downsamples data.
+
+The following example creates a notebook that queries **Coinbase bitcoin price
+sample data**, downsamples the data into hourly summaries, and then writes the
+downsampled data to an InfluxDB bucket.
+
+{{% note %}}
+If you do not have an existing bucket to write the downsampled data to,
+[create a new bucket](/influxdb/v2.1/organizations/buckets/create-bucket/).
+{{% /note %}}
+
+1. [Create a new notebook](/influxdb/v2.1/notebooks/create-notebook/).
+2.  In the **Build a Query** cell:
+
+    1.  In the **FROM** column under **{{% caps %}}Sample{{% /caps %}}**,
+        select **Coinbase bitcoin price**.
+    2.  In the next **FILTER** column, select **_measurement** from the drop-down list
+        and select the **coindesk** measurement in the list of measurements.
+    3.  In the next **FILTER** column, select **_field** from the drop-down list,
+        and select the **price** field from the list of fields.
+    4.  In the next **FILTER** column, select **code** from the drop-down list,
+        and select a currency code from the list.
+
+3.  Click {{% icon "notebook-add-cell" %}} after your **Build a Query** cell to 
+    add a new cell and select **{{% caps %}}Flux Script{{% /caps %}}**.
+
+4.  In the Flux script cell:
+    
+    1.  Use `__PREVIOUS_RESULT__` to load the output of the previous notebook
+        cell into the Flux script.
+    3.  Use [`aggregateWindow()`](/{{< latest "flux" >}}/stdlib/universe/aggregateWindow/)
+        to window data into one hour intervals and return the average of each hour.
+        Specify the following parameters:
+
+        - **every**: Window interval _(should be less than or equal to the selected query time range)_.
+        - **fn**: [Aggregate]({{< latest "flux" >}}/function-types/#selectors)
+          or [selector function]({{< latest "flux" >}}/function-types/#selectors)
```

Comment:
```suggestion
          or [selector]({{< latest "flux" >}}/function-types/#selectors) function
```

Reply:
curious, is something off with latest flux tag? routing to this on localhost:
/influxdb/cloud/notebooks/downsample/flux/v0.x/function-types/#selectors

same with curious, is something off with latest flux tag? routing to this on localhost:
/influxdb/cloud/notebooks/downsample/flux/v0.x/function-types/#aggregates

Reply:
Yeah, I missed the first `/` before the shortcode. I fixed them.